src/main/java/com/browserstack/automate/ci/common/tracking/PluginsTracker.java

```diff
@@ -57,6 +65,31 @@ public void onResponse(Call call, Response response) throws IOException {
         });
     }
 
+    private void initializeClient() {
+
+        Proxy proxy = JenkinsProxySettings.getJenkinsProxy() != null ? JenkinsProxySettings.getJenkinsProxy() : Proxy.NO_PROXY;
+        if (proxy != Proxy.NO_PROXY) {
+            String username = JenkinsProxySettings.getUsername();
+            String password = JenkinsProxySettings.getPassword();
```

Comment:
`final` .... consider `final` as `const` of JS. or any value which won't be re-assigned further...

Reply:
can be resolved